exchanges/coinbasepro/coinbasepro_wrapper.go

```diff
@@ -1338,3 +1332,10 @@ func FormatAssetOutbound(a asset.Item) string {
 	}
 	return a.Upper()
 }
+
+// LoadAlias returns the aliases for a currency pair, with the original pair included
+func (a *aliasStruct) LoadAlias(p currency.Pair) currency.Pairs {
```

Comment:
Good start, all the other references to Alias struct can be methods as suggested above. Can change name of aliasstruct to `productAlias` or `pairAlias` and have specific tests associated for the methods.